package.json

```diff
@@ -34,7 +34,7 @@
     "express-graphql": "^0.4.4",
     "graphiql": "^0.3.1",
     "graphql": "^0.4.13",
-    "lodash": "^3.10.1",
+    "lodash": "^4.3.0",
```

Comment:
Updated to lodash 4 to get `uniqBy`—there's a [lot of breaking changes](https://github.com/lodash/lodash/wiki/Changelog#v400), but I tried to search the project to cover them. Tests pass, but happy to punt and make this a separate PR (`uniq` in 3.0 would work fine).